bisect_ppx.opam

```diff
@@ -23,8 +23,7 @@ depends: [
   "cmdliner" {>= "1.0.0"}
   "dune" {>= "2.7.0"}
   "ocaml" {>= "4.02.0"}
-  "ocaml-migrate-parsetree" {>= "1.7.0" & < "2.0.0"}
-  "ppx_tools_versioned" {>= "5.4.0"}
+  "ppxlib" {>= "0.21.0"}
```

Comment:
What is the minimum version with which the non-BuckleScript PPX can be made to work? Is it only the registration interface that the non-bsb code is using right now? Can it be downgraded to loosen this ppxlib constraint?

Reply:
The code is [accessing the input name through Ppxlib](https://github.com/aantron/bisect_ppx/pull/327/files/6ad06d712cf9f3c96c3638910f0b35a314b1de48#diff-4bdd650c927dc010be2484e34719df1b48a1c6debc9f8bb81e97f822ae3d7de6R1585) for which points 4 and 5 of the [Ppxlib 0.21.0 release notes](https://github.com/ocaml-ppx/ppxlib/releases/tag/0.21.0) are needed. How important is it to loosen the ppxlib constraint?

Reply:
Ah, thanks for reminding me of that. It's more important to have access to the input name than to loosen the constraint.